consts/colors.js

```diff
@@ -1,6 +1,8 @@
 export default {
   primaryColor: '#29ab87',
+  accentColor: '#4629ab',
```

Comment:
Tried to introduce an accent purple color, only using it on the button for the TriviaQuestions component. At a certain point it might just be better to get an actual UI/UX person - I know there a couple out of TDC here

Reply:
Looks fine to me? lol I agree maybe someone has some time to give us notes. Much better than what I had lol